runtime/vm/OutOfLineINL_openj9_internal_foreign_abi_UpCallMHMetaData.cpp

```diff
@@ -41,17 +41,11 @@ OutOfLineINL_openj9_internal_foreign_abi_UpcallMHMetaData_resolveUpcallDataInfo(
 	J9ROMClass * jclROMClass = jclConstantPool->ramClass->romClass;
 	U_32 * cpShapeDescription = J9ROMCLASS_CPSHAPEDESCRIPTION(jclROMClass);
 #if JAVA_SPEC_VERSION >= 20
-	const int cpEntryNum = 10;
+	const int cpEntryNum = 8;
 #else /* JAVA_SPEC_VERSION >= 20 */
-	const int cpEntryNum = 12;
+	const int cpEntryNum = 10;
 #endif /* JAVA_SPEC_VERSION >= 20 */
```

Comment:
We should let the compiler count the number of entries:
```c
	static const U_16 cpIndex[] = {
...
```
and adjust the loop
```c
	for (int i = 0; i < (sizeof(cpIndex) / sizeof(cpIndex[0])); i++) {
...
```

Reply:
Agreed and updated as suggested above.